packages/packages/commonwell-sdk/src/client/commonwell.ts

```diff
@@ -346,13 +356,66 @@ export class CommonWell {
    */
   async deletePatient(meta: RequestMetadata, id: string): Promise<void> {
     const headers = await this.buildQueryHeaders(meta);
-    await this.api.delete(`/org/${this.oid}/patient/${id}/`, {
+    await this.api.delete(`${CommonWell.ORG_ENDPOINT}/${this.oid}/patient/${id}/`, {
       headers,
     });
 
     return;
   }
 
+  //--------------------------------------------------------------------------------------------
+  // Link Management
+  //--------------------------------------------------------------------------------------------
+
+  async upgradeOrDowngradePatientLink(meta: RequestMetadata, href: string): Promise<NetworkLink> {
+    const headers = await this.buildQueryHeaders(meta);
+    const resp = await this.api.post(
+      href,
+      {
+        proxy: {},
```

Comment:
let's define a data model for this, and make it an optional param in the signature, for reference:

```
"proxy":  {
        "relationship": "Parent",
        "name": "JohnSmith"
    }
```